### PR TITLE
Links

### LINKED ISSUE #1683: Links (model, table, entity)
Provide data modelling in BEdita/Core for "links".
Links extend object. They should have also the following properties:

 - http status
 - last update date

### PR DESCRIPTION
**Pull request by didoda**
This PR fixes #1683 
Adding:

 - migration for links table
 - link entity
 - links table
 - basic tests

### CODE REVIEW

File: plugins/BEdita/Core/config/Migrations/20200429070603_LinksTable.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        // links
+        $this->table('links', ['id' => false])
+            ->addColumn('id', 'integer', [
+                'default' => null,
+                'limit' => 10,
+                'null' => false,
+                'signed' => false,
+            ])
+            ->addPrimaryKey(['id'])
+            ->addColumn('http_status', 'string', [
```
Comment (stefanorosanelli) [inline plugins/BEdita/Core/config/Migrations/20200429070603_LinksTable.php:28]: `'url'`(or `uri`) field is missing?

File: plugins/BEdita/Core/config/Migrations/20200429070603_LinksTable.php [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            [
+                'create' => [
+                    'object_types' => [
+                        [
+                            'name' => 'links',
+                            'singular' => 'link',
+                            'description' => 'Links model',
+                            'plugin' => 'BEdita/Core',
+                            'model' => 'Links',
+                        ],
```
Comment (stefanorosanelli) [inline plugins/BEdita/Core/config/Migrations/20200429070603_LinksTable.php]: add also:
```
'core_type' => 1,
'enabled' => 0,
```

**Review by stefanorosanelli — CHANGES_REQUESTED** [review]

File: plugins/BEdita/Core/config/Migrations/20200429070603_LinksTable.php [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                'signed' => false,
+            ])
+            ->addPrimaryKey(['id'])
+            ->addColumn('http_status', 'string', [
+                'comment' => 'HTTP status',
+                'default' => null,
+                'limit' => null,
+                'null' => true,
+            ])
+            ->addColumn('last_update', 'date', [
```
Comment (stefanorosanelli) [inline plugins/BEdita/Core/config/Migrations/20200429070603_LinksTable.php]: `timestamp` or `datetime` maybe better

File: plugins/BEdita/Core/config/Migrations/20200429070603_LinksTable.php (hunk 1 above)
Comment (didoda) [reply plugins/BEdita/Core/config/Migrations/20200429070603_LinksTable.php:28]: yes... coming soon

**Review by stefanorosanelli — APPROVED** [review]

**Pull request merged**